prose.md

```diff
@@ -222,42 +223,42 @@ styling, and validation:
  - {varnish} to host the CSS, JS, and HTML templates
  - {pegboard} to serve as a validator (and converter!) for lesson content
 
-When thinking about the lesson structure itself, we thought about what would
-make sense for a maintainer or contributor when they saw the source files of
-a website. We created a a folder structure that is reflected in the website
+When thinking about how that compose the lessons should be organized,
+We created a folder structure that reflects the website
```

Comment:
I think this transition is a bit cumbersome in both versions. :thinking: what is the best way to convey that we are focusing on the way people view the lesson itself?